### DISCUSSION TITLE
(Project beta) Allow image upload in item body field

### DISCUSSION THREAD

**Discussion by lemeurherve** [community/community#9746] [Projects and Issues]
Unlike in the issues, trying to paste an image in project item body field doesn't generate an upload and the associated markdown.

It would be great to be able to.

As a workaround, it's still possible to open any issue, paste the image here and retrieve the generated markdown.

**Comment by borekb** [comment]
This is now possible but not with the same UX as in issues (high-res images are uploaded as plain Markdown `![image]()` instead of more "correct" `<img width="" src="">`), see https://github.com/github-community/community/discussions/19965.
